code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -12,6 +12,7 @@ Applications: 8 servants, 3 caches, and 100 CV
 	var/name = "scripture"
 	var/desc = "Ancient Ratvarian lore. This piece seems particularly mundane."
 	var/list/invocations = list() //Spoken over time in the ancient language of Ratvar. See clock_unsorted.dm for more details on the language and how to make it.
+	var/chanting //If the invocation's words are being spoken
```

Comment:
do not initialize numerical vars as null

Reply:
It's a boolean; it only needs to check if it exists or not.

Reply:
`var/chanting = FALSE`

Reply:
That doesn't change my point, and two other variables in this exact datum (`invoker` and `slab`, which also check for existence) function in the exact same way.

Reply:
those aren't booleans.
`var/chanting = FALSE`